themes/default/components/src/components/contact-us-form/test/contact-us-form.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('contact-us-form', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pulumi-contact-us-form items="[]"></pulumi-contact-us-form>');
+
+    const element = await page.find('pulumi-contact-us-form');
+    expect(element).toHaveClass('hydrated');
+  });
+});
```

Comment:
We would need to load in the HubSpot SDK to actually load the forms. I'm not super inclined to do that inside of the stencil component, just to avoid leaking secrets. I'd rather we opt for some level on integration testing on the website to check that things are working properly.